index.html

```diff
@@ -35,10 +35,17 @@ <h1 class="info-title">Comodidades</h1>
     </div>
     <div id="carrouselContainer" class="carrousel-container">
       <div class="carrousel">
-        <img src="assets/images/IMG_8056.JPG" class="carrousel__img"/>
+        <img src="assets/images/IMG_8056.JPG" class="carrousel__img"/ id="img1">
         <img src="assets/icons/arrow.svg" class="carrousel__arrow carrousel__arrow--left" data-direction="back"/>
         <img src="assets/icons/arrow.svg" class="carrousel__arrow carrousel__arrow--right" data-direction="ahead"/>
       </div>
+      <div class="pointer-list" id="divBtn">
+        <div class="pointer-list__point oval oval-gray" id="divBtn1" ></div>
+        <div class="pointer-list__point oval oval-gray" id="divBtn2" ></div>
+        <div class="pointer-list__point oval oval-gray" id="divBtn3" ></div>
+        <div class="pointer-list__point oval oval-gray" id="divBtn4" ></div>
+      </div>
+      <button href="#" class="button button--medium button--secondary">Ver galeria</button>
```

Comment:
I don't know if it's important, but the button component was upgraded and the classes aren't called that anymore.

Reply:
This will be true if the button component PR gets 2 approves before this one.